cairo1-run/src/main.rs

```diff
@@ -475,6 +475,54 @@ mod tests {
         assert_matches!(run(args), Ok(Some(res)) if res == expected_output, "Program {} failed with flags {}", program, extra_flags.concat());
     }
 
+    // these tests a separated so as to run them without --append_return_values and --proof_mode options
```

Comment:
small typo
```suggestion
    // these tests are separated so as to run them without --append_return_values and --proof_mode options
```

Reply:
I'm not very familiar with the cairo-vm. Why can't these tests be executed with `--append_return_values` or `--proof_mode`?

Reply:
Because to compile cairo programs with those options, we are required to use the `SquashedFelt252Dict` type. Since the added tests are using `Felt252Dict` instead, we need to remove those options.

Reply:
Done!